hera_qm/xrfi.py

```diff
@@ -1381,16 +1389,18 @@ def chi_sq_pipe(uv, alg='zscore_full_array', modified=False, sig_init=6.0,
 
 
 def xrfi_run(ocalfits_file=None, acalfits_file=None, model_file=None, data_file=None,
-             omnical_median_filter=True, abscal_median_filter=True,
-             omnical_chi2_median_filter=True, abscal_chi2_median_filter=True,
-             omnical_mean_filter=True, abscal_mean_filter=True,
-             omnical_chi2_mean_filter=True, abscal_chi2_mean_filter=True,
-             omnical_zscore_filter=True, abscal_zscore_filter=True,
-             omnivis_median_filter=True, data_median_filter=False,
-             omnivis_mean_filter=True, data_mean_filter=True,
-             correlations='both', history=None,
+             omnical_median_filter=True, omnical_mean_filter=True,
+             omnical_chi2_median_filter=True, omnical_chi2_mean_filter=True,
+             omnical_zscore_filter=True,
+             abscal_median_filter=True, abscal_mean_filter=True,
+             abscal_chi2_median_filter=True, abscal_chi2_mean_filter=True,
+             abscal_zscore_filter=True,
+             omnivis_median_filter=True, omnivis_mean_filter=True,
+             auto_median_filter=True, auto_mean_filter=True,
+             cross_median_filter=False, cross_mean_filter=True,
+             history=None,
              xrfi_path='', kt_size=8, kf_size=8, sig_init=5.0, sig_adj=2.0,
-             ex_ants=None, ant_str=None, metrics_file=None,
+             ex_ants=None, metrics_file=None,
              output_prefix=None, clobber=False,
              run_check=True, check_extra=True, run_check_acceptability=True):
     """Run the xrfi excision pipeline used for H1C IDR2.2.
```

Comment:
This line is no longer true. Maybe "originally developed for H1C IDR2.2" or something like that

Reply:
which part of this line are you referring too?

Reply:
Weird, I meant to reference the line `Run the xrfi excision pipeline used for H1C IDR2.2.` but it seems to have gotten misattributed.

Reply:
I'll just fix it myself.